samples/HelloDTFx/HelloDTFx/HelloDTFx.csproj

```diff
@@ -8,8 +8,11 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.DurableTask.Netherite" Version="1.4.2" />
-    <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="6.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Console" />
+  </ItemGroup>
+
+  <ItemGroup>
+	  <ProjectReference Include="..\..\..\src\DurableTask.Netherite\DurableTask.Netherite.csproj" />
```

Comment:
The samples were designed to use the published package, not the locally built one. I suppose you are changing this so it works with centralized package management?

I am not sure what the reason was to use the published package. Probably so that the sample runs standalone and users can copy/paste it easily. That will of course no longer work with the centralized package management. But it's probably not a big issue.

Reply:
I can switch it back definitely. I thought we should make samples built on the latest Netherite and thus use the locally built one.

Reply:
Updated. @sebastianburckhardt  As for the build version conflict warning of package "Microsoft.Bcl.AsyncInterfaces", is not becasue of the CPM, but because `Faster.Core` 2.6.5 is using v7.0.0, while other packages like `Azure.Core`,  sample packages `Microsoft.Extensions.Logging.Console` are still using v6.0.0.  I updated my PR to ping to use "Microsoft.Bcl.AsyncInterfaces" v7.0.0. to remove the warnings.